webview.h

```diff
@@ -2145,6 +2212,13 @@ class win32_edge_engine {
     return ok;
   }
 
+  void on_dpi_changed(int x, int y, const RECT &suggested_bounds) {
```

Comment:
To keep things simple, trust Windows to give us the appropriate new bounds. This is how it should be done according to the [sample for WM_DPICHANGED](https://learn.microsoft.com/en-us/windows/win32/hidpi/wm-dpichanged).

Reply:
Never mind that—we don't get accurate bounds.